themes/Backend/ExtJs/backend/order/controller/document.js

```diff
@@ -60,13 +60,29 @@ Ext.define('Shopware.apps.Order.controller.Document', {
     },
 
     /**
-     * Loads a new mail and open a new mail window
+     * Opens the window to send an email with a selected document to the customer.
+     *
+     * @param { Shopware.apps.Order.model.Receipt } record
      */
     openMail: function(record) {
-        var me = this,
-            order = me.getDocumentWindow().record;
+        var me = this;
```

Comment:
Now you just need to replace all occurrences of `this` in this method with `me` again...